crates/api_common/src/person.rs

```diff
@@ -443,3 +445,24 @@ pub struct VerifyEmail {
 #[cfg_attr(feature = "full", ts(export))]
 /// A response to verifying your email.
 pub struct VerifyEmailResponse {}
+
+#[skip_serializing_none]
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// Get posts that belong to a community followed by the person.
+pub struct GetPosts {
+  pub sort: Option<SortType>,
+  pub page: Option<i64>,
+  pub limit: Option<i64>,
+  pub unread_only: Option<bool>,
+  pub auth: Sensitive<String>,
+}
```

Comment:
There is already a struct with this name so it should be renamed. Or maybe it could be implemented simply as a boolean parameter on the existing posts endpoint.